README.md

```diff
@@ -9,14 +9,29 @@ docker build --file src/ros/mardan/docker/mardan_base_noetic.dockerfile --tag "m
 ## Development using base container
 
 ```bash
-docker run -it --rm --mount type=bind,source="$(pwd)"/src/ros/,target=/root/catkin_ws/src/ mardan_base_noetic:local /bin/bash
+docker run -it --rm --mount type=bind,source="$(pwd)"/src/ros/,target=/root/catkin_ws/src/ -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix mardan_base_noetic:local /bin/bash
```

Comment:
do we should specify that this only works in Ubuntu? I mean, we are not being specific about the restrictions to use this kind of commands.

Reply:
Good catch, yes. This command only works on Linux systems. I will improve the documentation.